packages/module/src/controller/QuickStartFooter.tsx

```diff
@@ -117,11 +120,31 @@ const QuickStartFooter: React.FC<QuickStartFooterProps> = ({
     [status, SecondaryButtonText, onRestart, taskNumber, totalTasks],
   );
 
+  const getToggleAlwaysShowTaskReview = React.useMemo(
```

Comment:
I don't think that there should not be a new toggle in the drawer, and it doesn't have to be user configurable.
The developer should be able to set the prop on the context or the `QuickStartContainer`.
i.e. [within this context object](https://github.com/patternfly/patternfly-quickstarts/blob/main/packages/dev/src/App.tsx#L93) or [through the newer QuickStartContainer](https://github.com/patternfly/patternfly-quickstarts/blob/main/packages/dev/src/App2.tsx#L98)

Reply:
This would be useful to expose on the demo app though, would be nice to change some settings on the fly to play and test the options. Perhaps a button in the demo header that when pressed could give you a preferences modal to set the current language, and this new setting, to start (and remove the `Home` route and page.

Reply:
I realize in my issue comment i mentioned the user being able to toggle the setting, i worded it poorly, what i meant was that the developer should be able to change the component prop or context value in the code to turn on the auto-show ability.